server/index.js

```diff
@@ -64,7 +144,7 @@ app.use((err, req, res, next) => {
   }
 });
 
-app.listen(process.env.PORT, () => {
+app.listen(3000, () => {
```

Comment:
Please do not hard-code the port. It should be defined in your `.env` file

Reply:
Fixed